packages/realm/src/Realm.ts

```diff
@@ -132,9 +156,12 @@ export class Realm {
 
   /**
    * Get the current schema version of the Realm at the given path.
-   * @param  {string} path
-   * @param  {any} encryptionKey?
-   * @returns number
+   * @param path - The path to the file where the
```

Comment:
Not at a computer now, but I'm curious if the dash after param name is ignored? If not, it should probably go away.

Reply:
The dash does get ignored but It could be good to make this consistent anyways.